types/af-utils__react-table/index.d.ts

```diff
@@ -0,0 +1,153 @@
+// Type definitions for @af-utils/react-table 0.0
+// Project: https://github.com/nowaalex/af-utils
+// Definitions by: Hunter Ross <https://github.com/huner2>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+import * as React from 'react';
+import * as Headless from 'af-utils__react-virtual-headless';
```

Comment:
Should that module be also re-exported?
https://github.com/nowaalex/af-utils/blob/master/packages/af-utils-react-table/src/index.js#L4

Reply:
I wasn't 100% sure how to handle that, I've now re-exported in both table and list.